.github/workflows/release.yml

```diff
@@ -24,11 +24,6 @@ jobs:
       - name: Set package version
         run: echo $(jq --arg v "${{ env.VERSION }}" '(.version) = $v' package.json) > package.json
 
-      - name: Setup Bun
-        uses: oven-sh/setup-bun@v1
-        with:
```

Comment:
Does it work without this?

Reply:
it does 

https://github.com/upstash/qstash-js/actions/runs/10771948555https://www.npmjs.com/package/@upstash/qstash/v/2.7.6-canary